### ISSUE TITLE
Change the page, modifies pageItems array twice.

### ISSUE THREAD

**Issue by guilhermewaess** [ErikSchierboom/knockout-paging#15]
When the page change, the pureComputed function re-evaluate pageItems twice, for example:

``` js
target.pageNumber(2);

triggers 2 events:

target.pageItems <- change with items from page 1 plus page 2.
target.pageItems <- change with items from page 2
```

This is not a big deal if i have few items on array, but, if a have many items this is a big problem: Array with 20000 items

``` js
taget.toLastPage() <- went to last page with no problems

taget.toFirstPage()
triggers 2 events:

target.pageItems <- change with all itens on array again (20000), and the screen will render 20000 items on screen) 
target.pageItems <- change with items from page 1
```

For fast solution on my project, i just insert a rateLimit: 1 on pageItems observable, this ensures that pageItems notifies subscribers only once. I don't know if this is a final solution, but I would be happy to help analyze and solve with a pr :D

**Comment by ErikSchierboom** [comment]
I'd be more than happy to accept a PR! Great analysis so far.

**Comment by guilhermewaess** [comment]
https://github.com/ErikSchierboom/knockout-paging/pull/16 :D

**Comment by ErikSchierboom** [comment]
Closed by #16.
